### ISSUE TITLE
Loopback 3

### ISSUE THREAD

**Issue by zbarbuto** [strongloop/loopback-sandbox#13]
Add a branch (or make master) use Loopback 3.

- [ ] Bug
- [x] Feature request

### Description of feature (or steps to reproduce if bug)

Now that loopback 3 is released, would be good to have a sandbox/demo project that uses the latest and greatest.

**Comment by superkhau** [comment]
@zbarbuto Agreed, bumping priority on this. Would you like to a submit a PR? Two tasks:

- [ ] master should have 3.x code scaffolded
- [ ] 2.x branch should have 2.x code scaffolded

**Comment by zbarbuto** [comment]
Unlikely I'd get to this personally.
